exonum-java-binding-core/src/main/java/com/exonum/binding/blockchain/serialization/TransactionResultSerializer.java

```diff
@@ -16,13 +16,18 @@
 
 package com.exonum.binding.blockchain.serialization;
 
+import static com.exonum.binding.blockchain.TransactionResult.MAX_USER_DEFINED_ERROR_CODE;
+
 import com.exonum.binding.blockchain.TransactionResult;
 import com.exonum.binding.common.serialization.Serializer;
 import com.google.protobuf.InvalidProtocolBufferException;
 
 public enum TransactionResultSerializer implements Serializer<TransactionResult> {
   INSTANCE;
 
+  private static final int SUCCESSFUL_RESULT_STATUS_CODE = 256;
+  private static final int UNEXPECTED_ERROR_CODE = 257;
```

Comment:
The same - `private static final int UNEXPECTED_ERROR_CODE = SUCCESSFUL_RESULT_STATUS_CODE + 1;`?